ui/app/components/empty-state.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```

Comment:
So I think we should extend OuterHTML like `block-empty` was doing and then move all of the `classNames` stuff to a div in the template. I think we should try to do that for new components so we don't have a mix of class stuff in and out of the component definition (more than we already do 😭).